assets/components/doubleHeading/doubleHeading.jsx

```diff
@@ -1,11 +1,21 @@
+// @flow
+
 // ----- Imports ----- //
 
 import React from 'react';
 
 
+// ---- Types ----- //
+
+type PropTypes = {
+  heading: string,
+  subheading: ?string,
```

Comment:
After an offline discussion, we decided to leave this component without change now and add a new card to the backlog to kill one element components like `SimpleHeading` in the future.